consent-protocol/tests/test_kai_voice_contract.py

```
@@ -0,0 +1,561 @@
+ROOT = Path(__file__).resolve().parents[1]
+if "hushh_mcp.services" not in sys.modules:
+    services_pkg = types.ModuleType("hushh_mcp.services")
+    services_pkg.__path__ = [str(ROOT / "hushh_mcp" / "services")]
+    sys.modules["hushh_mcp.services"] = services_pkg
+
+from hushh_mcp.services.voice_intent_service import (
+    VoiceIntentService,
+    _ALLOWED_COMMANDS,
+    _ALLOWED_TOOL_NAMES,
+    _UNCLEAR_STT_MESSAGE,
+)
```

Ruff (0.15.5)

[error] Module level import not at top of file

(E402)